internal/app/connectconformance/results.go

```diff
@@ -36,7 +36,7 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
-const timeoutCheckGracePeriodMillis = 500
+const timeoutCheckGracePeriodMillis = 100
```

Comment:
Give the tolerances in actual timeout tests are on the order of 100 millis, the value here seemed egregiously generous. So I tightened it down to 100 milliseconds. This exists because some implementations (including connect-go) start the timeout clock when a deadline is configured, not when the RPC is actually sent.

So this grace period (admittedly still a bit generous) allows up to 100 milliseconds to elapse in the client implementation between the time the timeout is configured and the time it is written into RPC header metadata.